internal/contributebot/README.md

```diff
@@ -13,3 +13,43 @@ incoming events are bursty.
 
 [github-async]: https://developer.github.com/v3/guides/best-practices-for-integrators/#favor-asynchronous-work-over-synchronous
 [github-ratelimit]: https://developer.github.com/v3/#rate-limiting
+
+## Setup
+
+1.  [Create a new GCP project][].
+1.  Set your project using `gcloud config set project PROJECTID`, where
+    `PROJECTID` is the project's ID.
+1.  Enable App Engine with `gcloud app create`.
```

Comment:
Consider using 

```bash
```

for this, with environment variables for the variables?

E.g.,

```bash
# Set your project id here.
PROJECTID=
# Set your bucket name for Terrafrom state here.
TERRAFORM_BUCKET_NAME=

gcloud config set project ${PROJECTID}
gcloud app create
gsutil mb -c regional -l us-west2 gs://${TERRAFORM_BUCKET_NAME}
...
```